web/concrete/src/Page/Type/Type.php

```diff
@@ -1123,15 +1118,10 @@ public function createDraft(PageTemplate $pt, $u = false)
 
         // we have to publish the controls to the page. i'm not sure why
         $controls = PageTypeComposerControl::getList($this);
-        $outputControls = array();
         foreach ($controls as $cn) {
-            $cn->publishToPage($p, array(), $controls);
+            $cn->publishToPage($p, array('name' => null, 'description' => null,'url_slug' => null), $controls);
         }
 
-        // now we need to clear out the processed controls in case we
-        // save again in the same request
-        CorePagePropertyPageTypeComposerControl::clearComposerRequestProcessControls();
-
```

Comment:
That method only interacts with itself statically so it would be safe to change it to be static IMO.